.azure-pipelines/azure-pipelines-live-test-run.yml

```diff
@@ -0,0 +1,51 @@
+# This pipeline
+# 1. Runs manually
+# 2. Makes sure master branch is in healthy state (run live tests)
+
+trigger: none
+
+pr: none
+
+jobs:
+
+- job: 'Run_Test_Ubuntu'
+  pool:
+    vmImage: 'Ubuntu-16.04'
+  
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '$(python.version)'
+
+  - task: PowerShell@2
+    displayName: 'Setup test machine'
+    inputs:
+      targetType: 'filePath'
+      filePath: 'scripts/setupCiMachine.ps1'
+
+  # Run a Python script.
+  - task: PythonScript@0
```

Comment:
should we put this step in runTests.ps1?
reason being that in case I want to run the tests in live mode locally (against my org)
then I will have to do multiple steps..
so putting this as part of runtests.ps1 is looking more reasonable

Reply:
I want to  make all scripts python. So I do not want to add stuff to the powershell script.
If you still feel otherwise we can do that.